azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/exception/LoginFailureException.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.tools.auth.exception;
+
+public class LoginFailureException extends AzureLoginException {
+    private static final long serialVersionUID = -94207672112213624L;
+
+    public LoginFailureException(String message) {
+        super(message);
+    }
+
+}
```

Comment:
Is it really necessary for us to create these new exceptions whose only difference is the class name?
how about only create one/two for us, let us use different values of its properties to describe different cases?

Reply:
Rule 2:
![image](https://user-images.githubusercontent.com/17075863/103621881-dd243080-4f70-11eb-891e-a745cbda50ef.png)